src/trace/span-inferrer.ts

```diff
@@ -355,6 +355,37 @@ export class SpanInferrer {
     return new SpanWrapper(this.traceWrapper.startSpan("aws.sns", options), spanWrapperOptions);
   }
 
+  createInferredSpanForEBSQS(
```

Comment:
I think this method is not needed, since `createInferredSpanForEventBridge` diff is only in the signature type for `event`, and of course the name.

Reply:
Yeah I think you're right, same with the SNSSQS method.